components/learn_more_trial_modal/start_trial_btn.tsx

```diff
@@ -81,14 +95,16 @@ const StartTrialBtn = ({
             return formatMessage({id: 'start_trial.modal.loaded', defaultMessage: 'Loaded!'});
         case TrialLoadStatus.Failed:
             return formatMessage({id: 'start_trial.modal.failed', defaultMessage: 'Failed'});
+        case TrialLoadStatus.Embargoed:
+            return formatMessage({id: 'admin.license.trial-request.embargoed'});
```

Comment:
Could you include the defaultMessage here and needed values object here? As this same message is scattered throughout files, you could define a const wherever it make the most sense (maybe in `components/learn_more_trial_modal`) like:

```
const embargoedMessage = {
  id: 'admin.license.trial-request.embargoed',
  defaultMessage: 'We were unable to process the request due to limitations for embargoed countries. <link>Learn more in our documentation</link>, or reach out to legal@mattermost.com for questions around export limitations.'
  values: {
    link: (text: string) => <a href='https://mattermost.com/pl/limitations-for-embargoed-countries'>{text}</a>,
  },
}
```

Hooks can consume it like `formatMessage(embargoedMessage)` and components can consume it like `<FormattedMessage {...embargoadMessage} />`

Reply:
I misunderstood what you were suggesting before getting started, but ended up getting to a similar solution. 

I've added a single component that renders this message, exported it, and then used that everywhere it's needed so there's no duplication. Lmk what you think

Reply:
works for me!